packages/webpack/package.json

```diff
@@ -17,8 +17,8 @@
     "@angular-cli/ast-tools": "^1.0.0"
   },
   "peerDependencies": {
```

Comment:
Correct, but since this is a peerDep we don't want to limit users. `2.0.2` also works and some projects might be using it.